deploy.sh

```diff
@@ -0,0 +1,14 @@
+my_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $my_dir
+git branch -D gh-pages
+git checkout -B gh-pages
+npm install
+bower install
+shopt -s dotglob && git rm -rf --ignore-unmatch *
+git checkout master www/
+mv www/* ./
+rm -rf www
+git checkout master scss
+git add -A
+now="$(date)"
+git commit -m "Deploy by $(whoami): ${now}"
```

Comment:
Don't forget to leave people where they started.  `git checkout master`

Which reminds me.  Might be worth checking if you started in `master`  As is, this script will deploy whatever branch you start on.